page/changelog/1.5.md

```diff
@@ -33,11 +33,11 @@ Released on June 8, 2008
 ## New bugtracker and SVN project
 
 * Dedicated bugtracker with version targeting and categories on [http://ui.jquery.com/bugs](http://ui.jquery.com/bugs)
-* Dedicated project on google code, restructured and cleaner SVN trunk and branches ([http://code.google.com/p/jquery-ui](http://code.google.com/p/jquery-ui))
+* Dedicated project on google code, restructured and cleaner SVN trunk and branches ([https://github.com/jquery/jquery-ui](https://github.com/jquery/jquery-ui))
 
 ## New features and API changes
 * Hundreds of bugfixes, enhancements and performance improvements with many components completely rewritten
-* Unified API for all plugins (see [http://jquery.com/blog/2008/02/12/jquery-ui-15b-new-api-more-features-huge-performance-boost/](http://jquery.com/blog/2008/02/12/jquery-ui-15b-new-api-more-features-huge-performance-boost/))
+* Unified API for all plugins (see [https://jquery.com/blog/2008/02/12/jquery-ui-15b-new-api-more-features-huge-performance-boost/](https://jquery.com/blog/2008/02/12/jquery-ui-15b-new-api-more-features-huge-performance-boost/))
```

Comment:
This will result in a redirect which will take you back to HTTP anyway. The correct URL is https://blog.jquery.com/2008/02/12/jquery-ui-15b-new-api-more-features-huge-performance-boost/